net/pfSense-pkg-WireGuard/files/usr/local/www/wg/vpn_wg_tunnels_edit.php

```diff
@@ -289,7 +289,7 @@
 			'fa-trash'
 		))->addClass('btn-warning btn-sm');
 	
-		$section->add($group);		
+		$section->add($group);	
```

Comment:
There is still some trailing whitespace on this line

Reply:
Fixed.